torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5547,29 +5536,41 @@ def sample_inputs_svd(op_info, device, dtype, requires_grad=False, **kwargs):
     make_arg = partial(make_fullrank, dtype=dtype, device=device, requires_grad=requires_grad)
 
     is_linalg_svd = (op_info.name == "linalg.svd")
+    batches = [(), (0, ), (3, )]
+    ns = [0, 3, 5]
 
-    batches = [(), (0, ), (2, )]
-    ns = [0, 2, 5]
-
-    # The .abs() is to make these functions invariant under multiplication by e^{i\theta}
-    # since the complex SVD is unique up to multiplication of the columns of U / V a z \in C with \norm{z} = 1
-    # See the docs of torch.linalg.svd for more info
-    def check_grads(usv):
+    def uniformize(usv):
         S = usv[1]
         k = S.shape[-1]
         U = usv[0][..., :k]
         Vh = usv[2] if is_linalg_svd else usv[2].mH
         Vh = Vh[..., :k, :]
-        return (U.abs(), S, Vh.abs())
+        return U, S, Vh
```

Comment:
Do you have a test which checks the error message when the objective function does not satisfy the invariant?

Reply:
I do not, as these tests are rather brittle, as the errors might change and so on. Also, testing an error in the backward within current OpInfos might be tricky.

Reply:
We can do so in `test_linalg.py`. Looks like just summing up the inputs will break the invariance, right?

Reply:
Sure, I'll add a test.